src/sdk.js

```diff
@@ -1,4 +1,26 @@
-const elements = document.querySelectorAll('div.likecoin-button-like');
+const style = document.createElement('style');
+style.innerHTML = `
```

Comment:
not sure if we have better way to do this instead of `style.innerHTML`

Reply:
Maybe we just minimize the CSS for now? We can parse SCSS/Stylus in Webpack later

Reply:
Is using [insertRule](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/insertRule) too obsolete, or is there something similar in modern es?

Reply:
That looks primitive. Maybe need to study other SDK for CSS injection